requirements.txt

```diff
@@ -40,10 +40,9 @@ tensorboardX>=1.6
 # Scale-up/out
 dask[complete]>=1.2.1
 
-# Note: you may want to remove PySpark from the 
-# required installs. Installing a new version may
-# overwrite your existing system install.
-pyspark>=2.4.3
+# Note: we don't include PySpark in the required installs.
+# Installing a new version may overwrite your existing system install.
+# pyspark>=2.4.3
```

Comment:
we don't want  == some version that works for the tests?

Reply:
yeah I could make a requirements-spark.txt file to pin it just for tests